modules/lvms-using-raid-arrays.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="lvms-using-raid-arrays_{context}"]
+= Integrating software RAID arrays with {lvms}
+
+Logical Volume Manager (LVM) does not support creating a software RAID. However, you can create the RAID array using the `mdadm` utility, and integrate the RAID array with {lvms}.
+You can integrate the RAID array with {lvms} while creating the `LVMCluster` custom resource (CR). 
+
+.Prerequisites
+
+* You created a software RAID during the {product-title} installation.
++
+[NOTE]
+====
+You can only create the RAID array during the installation of {product-title}.
+====
+
+* You have installed {lvms}.
+
+.Procedure
+
+. Open the `LVMCluster` custom resource (CR) YAML file.
++
+[NOTE]
+====
+If you have created the LVM cluster, then you can edit the existing `LVMCluster` CR.
```

Comment:
```suggestion
If you have created the LVM cluster, edit the existing `LVMCluster` CR.
```

IBM SG:

![Screenshot from 2024-01-31 11-19-36](https://github.com/openshift/openshift-docs/assets/57954076/f40ae07d-c5e1-43d3-bfff-fcbadefe6af3)

Reply:
Updated as per the suggestion.

Reply:
:heavy_check_mark: